lib/semian/sysv_shared_memory.rb

```diff
@@ -0,0 +1,75 @@
+module Semian
+  module SysVSharedMemory #:nodoc:
```

Comment:
Haven't had the opportunity to review this yet, but looks quite complicated at first glance